rocAL/rocAL/source/augmentations/color_augmentations/node_brightness.cpp

```diff
@@ -43,7 +43,7 @@ void BrightnessNode::create_node() {
     vx_scalar in_layout_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &input_layout);
     vx_scalar out_layout_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &output_layout);
     vx_scalar roi_type_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &roi_type);
-    _node = vxExtrppNode_Brightness(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), in_layout_vx, out_layout_vx, roi_type_vx);
+    _node = vxRppBrightness(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), in_layout_vx, out_layout_vx, roi_type_vx);
```

Comment:
@fiona-gladwin we need to add "ext" to all the tensor ones. Something like "vxExtRppBrightness" and similar. @rrawther is that okay with you, as discussed offline

Reply:
@rrawther and @LakshmiKumar23 Please let me know if we need to rename is as
`vxExtRppBrightness` / `vxExtRppNodeBrightness` / `vxExtRppNode_Brightness`
Since all OpenCV extenstions has the word node included in their prefix
`vxExtCvNode_add`

Reply:
@fiona-gladwin I would suggest using ```vxExtRppBrightness```